package-overrides/npm/bootstrap@3.3.7.json

```diff
@@ -0,0 +1,15 @@
+{
+  "main": "js/bootstrap",
+  "directories": {
+    "lib": "dist"
+  },
+  "shim": {
+    "js/bootstrap": {
+      "deps": [
+        "jquery",
+      ],
+      "exports": "$"
+    }
+  },
+  "files": [ "dist" ]
```

Comment:
Best to leave this out as well, no reason to filter out the other files if users may find them useful.